copier/main.py

```diff
@@ -1232,7 +1232,7 @@ def _apply_update(self) -> None:  # noqa: C901
                         # The 3-way merge might have resolved conflicts automatically,
                         # so we need to check if the file contains conflict markers
                         # before storing the file name for marking it as unmerged after the loop.
-                        with Path(fname).open() as conflicts_candidate:
+                        with Path(fname).open(encoding="utf-8") as conflicts_candidate:
                             if any(
                                 line.rstrip()
                                 in {"<<<<<<< before updating", ">>>>>>> after updating"}
```

Comment:
Or how about opening the file with `"rb"` mode and turning the markers into byte strings?

```suggestion
                        with Path(fname).open("rb") as conflicts_candidate:
                            if any(
                                line.rstrip()
                                in {b"<<<<<<< before updating", b">>>>>>> after updating"}
```